### DISCUSSION TITLE
Unable to give style to hotspot file

### DISCUSSION THREAD

**Discussion by jainAtish** [filipecorrea/react-image-hotspots#81] [Q&A]
Unable to access style passed as props

File: Hotspot.js

In document : style props is present 
Hotspot.propTypes = { 
 /** percentage from the left of the image to show this hotspot */  
x: _propTypes["default"].number,  
/** percentage from the top of the image to show this hotspot */  
y: _propTypes["default"].number,
  /** the content of the hotspot */  
content: _propTypes["default"].element
 /** the style of the hotspot */
  style: PropTypes.object
};

After Install : No Style props available nor able to access it
Hotspot.propTypes = { 
 /** percentage from the left of the image to show this hotspot */  
x: _propTypes["default"].number,  
/** percentage from the top of the image to show this hotspot */  
y: _propTypes["default"].number,
  /** the content of the hotspot */  
content: _propTypes["default"].element
};

Any upcoming version is coming fixing this?

**Comment by filipecorrea** [comment]
Hi @jainAtish. Style can be applied using the `Hotspot` content:

<img width="1345" alt="Screenshot 2023-03-10 at 12 56 17" src="https://user-images.githubusercontent.com/1496656/224310038-9f3fb86e-3e7b-4164-9490-ce04b8d339eb.png">

You can also test it using the [react-image-hotspots storybook](https://filipecorrea.github.io/react-image-hotspots/).
